src/components/Board.js

```diff
@@ -4,17 +4,31 @@ import Square from './Square';
 import PropTypes from 'prop-types';
 
 
-const generateSquareComponents = (squares, onClickCallback) => {
+
+const generateSquareComponents = (squares, onClickCallback, currentPlayer) => {
   // Complete this for Wave 1
   // squares is a 2D Array, but 
   //  you need to return a 1D array
   //  of square components
+  let squareArray = [];
+
+  squares.forEach((row) => {
+    row.forEach((square) => {
+      squareArray.push(
+      <Square
```

Comment:
Note the following warning. Make sure to include a `key` prop `key={square.id}`:
   
```
    console.error
      Warning: Each child in a list should have a unique "key" prop.
      
      Check the render method of `Board`. See https://reactjs.org/link/warning-keys for more information.
          at Square (/app/src/components/Square.js:30:12)
          at Board (/app/src/components/Board.js:30:18)
          
```